manuscript/ha-docker-swarm/nodes.md

```diff
@@ -63,6 +63,19 @@ Set your local timezone, by running:
 ln -sf /usr/share/zoneinfo/<your timezone> /etc/localtime
 ```
 
+### Install Docker
+
+A fresh installation of Ubuntu does not include Docker, so you will need to install it manually. Run the following command **on each node**
+
+```
+apt install docker.io -y
```

Comment:
I think we should explain what the ``docker.io`` package is and that it is not ``docker-ce`` and you can't go to their org to get support.

Reply:
Checking into this. I was not aware that `docker.io` package was different than `docker-ce`.

Reply:
After doing some research, the docker.io package is the name of the official Debian package and is the preferred way to install Docker on Debian. I don't think any further comment on it is necessary.